src/components/editor/EditAuthorsForm.jsx

```diff
@@ -26,59 +29,74 @@ export default class EditAuthorsForm extends BaseComponent {
   }
 
   render() {
+    const styles = {
+      wrapper: {
+        display: 'flex',
+        flexWrap: 'wrap',
+      },
+    }
+
+    const renderChips = (authors) =>
+      _.map(authors, (author) => {
+        return(
+          <Chip
+            key={author.id}
+            onRequestDelete={this.props.handleDeleteAuthor.bind(null, author.id, true)}
+            style={{margin: 4}}
+          >
+            {author.name}
+          </Chip>
+        )
+      });
+
     return (
-      <div style={this.props.style}>
+      <div>
         {
-          _.map(this.props.authors, (author) => {
-            // Striked will correctly evaluate to false if there is no key
-            // with that slug yet, which means it has neither been striked nor unstriked yet
-            // It will return undefined which is a falsy value
-            let striked = this.props.authorsDeleted[author.id];
-            let authorNameStyle = {marginLeft: "1em"};
-            if (striked) {
-              authorNameStyle.textDecoration = "line-through";
-              authorNameStyle.opacity = 0.5;
-            }
+          // TODO: EMIL: not exactly sure what this code is doing so I didnt want to delete
```

Comment:
This is what I thought it would do, but this functionality wasn't working on my computer when I was testing it. Maybe it was a browser specific thing?

The 'Update Author' section works a bit differently in this new refactor. I use 'chips' which can be added or deleted instead of striking out authors. I don't really understand the added value of the striking out thing when the authors can just be added back by searching.